wrappers.c

```diff
@@ -426,10 +428,10 @@ sighandler_t __unix_signal(int signum, sighandler_t handler)
         [16] = SIGUSR1,
         [17] = SIGUSR2,
         [18] = SIGCHLD,
-        [19] = SIGPWR,
+        [19] = SIGCONT,
```

Comment:
The intention here is to map UNIX signal numbers to Linux numbers, and it thinks 19 is SIGPWR. If FreeBSD doesn't have it, I think it should be -1, maybe just
```c
#ifdef SIGPWR
[19] = SIGPWR,
#else
[19] = -1,
#endif

Reply:
Hm, while I did quickly hack this by finding some supposedly portable list of magic numbers online, I'm not absolutely clear about the intention behind this lookup table, perhaps you could help clarify it.

SIGPWR seems to be a Linux-specific symbol, actually none of these is standard. Is the original code, or our extensions to it, registering a handler for signal 19?

Similarly, magic numbers in the table  (SIGUSR1, SIGUSR2, SIGCHLD...) that are not specific to Linux were likely the same on System V, so it eludes me why they would need to be translated. I can comment out all of these and the program seems to run fine, so I'm confused.

Finally, how did we arrive at translating the magic number 19 to the value of SIGPWR? On my Linux system, for example, SIGPWR is 30.